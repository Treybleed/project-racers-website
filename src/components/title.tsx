import React, { FunctionComponent } from "react"
import * as styles from  '../styles/home.module.scss';

type TitleProps = {
    title: string
  }

  export const Title: FunctionComponent<TitleProps> = ({ title }) => 
  <div className={styles.title}>
      {title}
  </div>