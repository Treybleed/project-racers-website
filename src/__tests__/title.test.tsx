import * as React from 'react'
import { shallow } from 'enzyme';
import { Title } from '../components/title';

describe('Title', () => {
  it('Renders Title component', () => {
    const title = shallow(<Title title="Test title"></Title>);
    expect(title).toMatchSnapshot();
  });
});