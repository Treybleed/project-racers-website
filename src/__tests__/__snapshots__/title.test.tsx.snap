// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Renders Title component 1`] = `ShallowWrapper {}`;
